src/pages/Dgfip.js

```diff
@@ -158,24 +160,85 @@ const DonneesDescription = () => (
     </table>
   </div>
 );
+const donneesDisponibles = [
+    {
+      "name": "dgfip_revenu_fiscal_de_reference_n_moins_1",
+      "humanName": "DGFIP - Revenu fiscal de référence (RFR) et nombre de parts (dernière année de revenu)"
+    },
+    {
+      "name": "dgfip_revenu_fiscal_de_reference_n_moins_2",
+      "humanName": "DGFIP - Revenu fiscal de référence (RFR) et nombre de parts (avant-dernière année de revenu)"
+    },
+    {
+      "name": "dgfip_adresse_fiscale_de_taxation_n_moins_1",
+      "humanName": "DGFIP - Adresse fiscale de taxation au 1er janvier (dernière année de revenu)"
+    },
+    {
+      "name": "dgfip_adresse_fiscale_de_taxation_n_moins_2",
+      "humanName": "DGFIP - Adresse fiscale de taxation au 1er janvier (avant-dernière année de revenu)"
+    }
+  ];
+
+// Le demandeur valide les modalités d'utilisation
+const CguDescription = () => (
+  <div className="information-text">
+    <p>
+      Votre raccordement à l‘API « impôt particulier » nécessite l‘acceptation
+      de la convention d‘adhésion fixant vos engagements et ceux de la DGFIP et
+      la DINSIC. Les liens ci-dessous vous permettront de visualiser la
+      convention type ainsi que ses annexes.
+    </p>
+  </div>
+);
+const cguLink = "/docs/API_impots_particulier_template_corps_juridique_avec_annexes.pdf"
+
+// Le demandeur doit remplir des contenus supplémentaires
+const ContenusSupplementaires = DgfipEntrantsTechniques;
```

Comment:
I like your idea of viewing this last part as an optional additional content.

Can we use an english variable name instead of `ContenusSupplementaires` ? (as it is not a business related name)